### PR TITLE
feat(ipfs): documentation about ipfs-naming project

### PR DESCRIPTION
**Pull request by OnsagerHe**
### Your checklist for this pull request

- [X] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Update `ipfs` folder names to `ipfs-pinning` and add concepts, how-to and quickstart for ipfs naming project

### CODE REVIEW

**Review by ldecarvalho-doc — APPROVED** [review]

File: labs/ipfs-naming/concepts.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+The IPNS name corresponds to the cryptographic hash of your public key. It is linked to a value or record denoted by either `/ipfs/cid` or `/ipns/key`. Each name has a constant lifespan and is subject to regular updates by the service. Additionally, the record can be modified by the holder of the private key, ensuring continuous verifiability.
+
+</Concept>
+
+<Concept>
+
+## Peer-to-peer (P2P)
+
+Architecture that partitions tasks or workloads between peers. Peers are equally privileged participants in the network. They form a peer-to-peer network of nodes.
+
```
Comment (bene2k1) [inline labs/ipfs-naming/concepts.mdx:61]: Just a thought but maybe add a description what is a kubo client to the concepts?

**Review by bene2k1 — APPROVED** [review]

File: labs/ipfs-pinning/how-to/add-pin.mdx [hunk 2]
```diff
@@ -19,22 +19,22 @@ Pinning a file essentially allows you to specify which particular content should
 
 When you pin your content using our Scaleway pinning services, you make sure it remains available and accessible on the IPFS network at all times. With your content now available on both your local node and our resilient and reliable external nodes, you drastically minimize the risk of disruptions thanks to redundancy. You can also enhance your data storage efficiency by pinning your content to your Scaleway node, and then remove it from your local node.
```
Comment (jcirinosclwy) [inline labs/ipfs-pinning/how-to/add-pin.mdx:20]:
```suggestion
When you pin your content using our Scaleway pinning services, you make sure it remains available and accessible on the IPFS network at all times. With your content now available on both your local node and our resilient and reliable external nodes, you drastically minimize the risk of disruptions thanks to redundancy. You can also enhance your data storage efficiency by pinning your content to your Scaleway node, and then removing it from your local node.
```

File: labs/ipfs-naming/how-to/import-key.mdx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    <Message type="important">
+      The key must be in `base64` format.
+    </Message>
+7. Click on **Download key** and choose the file containing your private key.
+    <Message type="note">
+      You can use `ipfs key export $NAME_KEY` to generate the file.
+    </Message>
+8. Enter a value for the keys.
+    <Message type="note">
+      The value must be an IPNS key or a CID.
```
Comment (nerda-codes) [inline labs/ipfs-naming/how-to/import-key.mdx:41]: maybe link to the concepts?

File: labs/ipfs-naming/concepts.mdx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+Data over IPFS is identified by its content, not by its location (like HTTP does), providing a new way to access data through a unique content identifier (CID). Consequently, data is immutable and verifiable, allowing any device to be involved in the network, enhancing the redundancy.
+
+</Concept>
+
+<Concept opened>
+
+## Content Identifier (CID)
+
+The content identifier is string of character that points to a content. The content can be available on the network, or not. The CID is composed from the cryptographic hash of the content. It is used to identify and fetch the content through the network.
```
Comment (jcirinosclwy) [inline labs/ipfs-naming/concepts.mdx:27]:
```suggestion
The content identifier is string of character that points to a content. The content can be available on the network, or not. The CID is composed of the cryptographic hash of the content. It is used to identify and fetch the content through the network.
```

**Review by jcirinosclwy — APPROVED** [review]

File: labs/ipfs-naming/how-to/update-name.mdx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+<Message type="requirement">
+    - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
+    - You have [created a key](/labs/ipfs-naming/how-to/create-key)
+</Message>
+
+
+1. Click **IPFS Naming** in the **Labs** section of the [Scaleway console](https://console.scaleway.com) side menu. The IPFS Naming section displays.
+2. Click on the resource name to access the record information page.
+3. Click on **Update Value** and set your new value
```
Comment (nerda-codes) [inline labs/ipfs-naming/how-to/update-name.mdx:26]:
```suggestion
3. Click **Update Value** and set your new value
```

**Review by nerda-codes — APPROVED** [review]

File: labs/ipfs-naming/concepts.mdx (hunk 1 above)
Comment (ldecarvalho-doc) [reply labs/ipfs-naming/concepts.mdx:61]: will be added later 🙏🏻

File: labs/ipfs-naming/how-to/import-key.mdx (hunk 3 above)
Comment (ldecarvalho-doc) [reply labs/ipfs-naming/how-to/import-key.mdx]: done

**Pull request merged**